src/Factories/FunFunFactory.php

```diff
@@ -922,48 +958,35 @@ private function newDonationConfirmationMailer(): DonationConfirmationMailer {
 					'mail_subject_confirm_donation',
 					'mail_subject_confirm_donation_promise'
 				)
-			)
-		);
-	}
-
-	public function newPayPalUrlGeneratorForDonations(): PayPalUrlGenerator {
-		return new PayPalUrlGenerator(
-			$this->getPayPalUrlConfigForDonations(),
-			$this->getPaymentProviderItemsTranslator()->trans( 'paypal_item_name_donation' )
-		);
-	}
-
-	public function newPayPalUrlGeneratorForMembershipApplications(): PayPalUrlGenerator {
-		return new PayPalUrlGenerator(
-			$this->getPayPalUrlConfigForMembershipApplications(),
-			$this->getPaymentProviderItemsTranslator()->trans( 'paypal_item_name_membership' )
+			),
+			$this->newGetPaymentUseCase()
 		);
 	}
 
 	private function getPayPalUrlConfigForDonations(): PayPalConfig {
 		return PayPalConfig::newFromConfig(
-			array_merge( $this->config['paypal-donation'], [ 'locale' => $this->getLocale() ] )
+			array_merge( $this->config['paypal-donation'], [ 'locale' => $this->getLocale() ] ),
+			new TranslatableDescriptionStub()
 		);
 	}
 
 	private function getPayPalUrlConfigForMembershipApplications(): PayPalConfig {
 		return PayPalConfig::newFromConfig(
-			array_merge( $this->config['paypal-membership'], [ 'locale' => $this->getLocale() ] )
+			array_merge( $this->config['paypal-membership'], [ 'locale' => $this->getLocale() ] ),
+			new TranslatableDescriptionStub()
```

Comment:
This method should be gone, we don't have paypal for memberships any more. Please delete if possible, or deprecate